hw1/02-palindrome.js

```diff
@@ -1,3 +1,46 @@
-const elem = document.querySelector('input');
+const inputElem = document.querySelector("input");
+const NOT_A_NUMBER = "Not a number.";
+const POSITIVES_ONLY = "Input cannot be less than 0.";
+const NOT_PALINDROME = "No. Try again";
+const IS_PALINDROME = "Yes! This number is a palindrome!";
 
-elem.addEventListener('input', handleInput);
+inputElem.addEventListener("input", handleInput);
+
+function inputVerify(input) {
+    const verifyResult = isNaN(input) === true || input.length === 0;
+    return verifyResult;
+}
+
+function dangerElem(message) {
+    const modElem = document.getElementById("input-result");
+    modElem.classList = "";
+    modElem.classList.add("text-danger");
+    modElem.textContent = message;
+    return;
+}
+
+function successElem(message) {
+    const modElem = document.getElementById("input-result");
+    modElem.classList = "";
+    modElem.classList.add("text-success");
+    modElem.textContent = message;
+    return;
+}
+
+function handleInput() {
+    const receivedInput = inputElem.value;
+    const inputReversed = Array.from(receivedInput).reverse().join("");
+
```

Comment:
Clever idea to make the user input into an array for later manipulation/reversal check.